README.md

```diff
@@ -58,6 +59,36 @@ class AdminController < ApplicationController
 end
 ```
 
+## Restricting access to mounted Middleware
+
+Add the gems to your Gemfile:
+
+```ruby
+# Use Balrog for authentication
+gem 'balrog'
+# Use Sidekiq for background jobs
+gem 'sidekiq'
+```
+
+Then disable Sidekiq Web's session in config/initializers/sidekiq.rb
```

Comment:
*IF* you are using Balrog to protect `Sidekiq::Web`, then you need to disable its own sessions. Link to issue/documentation about this.